Makefile

```diff
@@ -9,8 +9,9 @@ LDFLAGS = -L-lSDL
 NFD = nativefiledialog/src
 
 ifeq ($(OS),Windows_NT)
+EXEEXT = .exe
 cobjs += nfd_win.obj nfd_common.obj
-LDFLAGS += -L/SUBSYSTEM:WINDOWS
+LDFLAGS += -L-lmsvcrt -L/SUBSYSTEM:WINDOWS
```

Comment:
Not sure about this. It turns:

    nfd_win.obj : error LNK2019: unresolved external symbol __imp__wassert referenced in function _ZL18AddFiltersToDialogP11IFileDialogPKc
    nfd_common.obj : error LNK2019: unresolved external symbol __imp__assert referenced in function NFD_PathSet_GetCount

into:

    LINK : warning LNK4098: defaultlib 'libcmt' conflicts with use of other libs; use /NODEFAULTLIB:library
    LINK : warning LNK4098: defaultlib 'libvcruntime' conflicts with use of other libs; use /NODEFAULTLIB:library